### PR TITLE
Update Next.js icon and data

### LINKED ISSUE #7626: [Next.js] No brand guidelines
### Brand Name

Next.js

### Official Resources for Icon and Color

https://vercel.com/design#next-js

### Additional Comments

The brand guidelines aren't linked to the next.js icon

### Contributing

No

### LINKED ISSUE #7742: Update Next.js icon
### Brand Name

Next.js

### Official Resources for Icon and Color

https://vercel.com/design/brands#symbol

### Additional Comments

The new icon has color gradients, or rather fades to transparency, which cannot be represented in a monochromatic icon.

### Contributing

Yes

### PR DESCRIPTION
**Pull request by sisp**
image

**Issue:** closes #7742, closes #7626

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

The new Next.js wordmark uses gradient transitions which cannot be converted to a monochromatic icon. We've discussed in #7742 that it might be better to use the full wordmark instead. I've also updated the guidelines link as suggested in #7626.

### CODE REVIEW

**Comment by LitoMore** [comment]
I'd prefer just `N`.

[Next.js](https://github.com/vercel/next.js/blob/canary/packages/next/README.md?plain=1#L22) is using the logo on their main project. The full trademark looks not good on a badge.

**Comment by LitoMore** [comment]
Let's ask Next.js for help. Just let them  know Simple Icons does not support gradient colors.

It would be good if they can design a specific version for Simple Icon.

**Comment by sisp** [comment]
I've started a discussion over at Next.js: https://github.com/vercel/next.js/discussions/41193

**Comment by adamrusted** [comment]
Hey @ijjk - is there currently a monochrome (including without the gradient) version of the NextJS `N` icon? If we're able to use it here - any chance you can share somewhere it's publicly available?

**Comment by adamrusted** [comment]
Should we just include the full wordmark until we've heard back from them? That way, we're up to date, albeit with a non-ideal icon.

File: _data/simple-icons.json
Comment (adamrusted) [inline _data/simple-icons.json:7434]:
```suggestion
            "source": "https://vercel.com/design/brands#next-js",
            "guidelines": "https://vercel.com/design/brands#next-js"
```

**Review by adamrusted — COMMENTED** [review]
The SVG itself is a perfect match, if we go ahead with the full wordmark until Next update their icon format. Just a quick change to the JSON, seems they've updated URLs since this PR was put in. Once that's done, let's get this merged in!

**Comment by LitoMore** [comment]
I want to keep their icon visual at https://github.com/vercel/next.js.

BTW, the Next.js Conf will be back in October. They should revise the logo again.

**Comment by adamrusted** [comment]
> I want to keep their icon visual at https://github.com/vercel/next.js.

Oh I totally agree, but in the absence of an official monochrome treatment for the 'new' icon - we should go with a safe monochrome treatment from the brand. Let's hold fire until Next Conf, and if there's no significant re-brand I'd look to merge this one in.

**Comment by PratyushJayachandran** [comment]
> > I want to keep their icon visual at https://github.com/vercel/next.js.
> 
> Oh I totally agree, but in the absence of an official monochrome treatment for the 'new' icon - we should go with a safe monochrome treatment from the brand. Let's hold fire until Next Conf, and if there's no significant re-brand I'd look to merge this one in.

So I was looking their Readme and saw them using Next.js icon hosted in simple-icons.
It is the same as their favicon on https://nextjs.org/
So why are we changing it to wordmark?

**Review by adamrusted — CHANGES_REQUESTED** [review]
Still no official monochrome treatment from what I can see. We can _either_ proceed with the wordmark, as that's the only monochrome treatment offered by the brand at this time - or we adapt their currently used icon (as I imagine we had to do with the old version too!)

**Comment by PeterShaggyNoble** [comment]
I vote to give the icon our own treatment. But, then, I'll always vote in favour of an icon over a wordmark!

**Comment by PeterShaggyNoble** [comment]
So, @simple-icons/maintainers, where have we landed on this?

**Comment by service-paradis** [comment]
They have a monochrome version of their favicon buried in their Github:
https://github.com/vercel/next.js/blob/6b3d145e7e7e88037f4e326cc63038814ab5fd76/examples/cms-prepr/public/favicon/safari-pinned-tab.svg?plain=1#L4

So, I think we're safe!

**Comment by adamrusted** [comment]
@service-paradis - that looks to be the _old_ version of the icon. The new one is a much chunkier font.

**Comment by PeterShaggyNoble** [comment]
It's been well over a year since Lito started the discussion in their repo and there's been no response so I think it's time we make an executive decision. Consensus seems to be that we use the icon rather than the wordmark so could you update the PR accordingly, @sisp? The best option is probably just to use the same treatment as for our existing icon.

File: icons/nextdotjs.svg
Comment (adamrusted) [inline icons/nextdotjs.svg:1]: If we take the `icon` found at the new source URL, and apply the gaps in the same way we did with the previous one, we end up with:
image

I can submit this version if it'd be preferred?

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
I think the diagonal of the "N" should still touch the edge of the circle here, @adamrusted.

File: icons/nextdotjs.svg
Comment (adamrusted) [inline icons/nextdotjs.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Next.js</title><path d="M18.665 21.978C16.758 23.255 14.465 24 12 24 5.377 24 0 18.623 0 12S5.377 0 12 0s12 5.377 12 12c0 3.583-1.574 6.801-4.067 9.001L9.219 7.2H7.2v9.596h1.615V9.251l9.85 12.727Zm-3.332-8.533 1.6 2.061V7.2h-1.6v6.245Z"/></svg>
```

**Review by PeterShaggyNoble — APPROVED** [review]

**Pull request merged**
